cypress/integration/views/credentials.view.ts

```diff
@@ -18,3 +18,8 @@ export const descriptionInput = "input[name='description']";
 export const usernameInput = "input[name=user]";
 export const passwordInput = "input[name=password]";
 export const privatePassphraseInput = "input[aria-label='Private Key Passphrase']";
+export const modal = "[id^=pf-modal-part-]";
+export const navLink = ".pf-c-nav__link";
+export const createBtn = "#create-credential-button";
+export const closeNotification = "button[aria-label='close-notification']";
+export const confirmButton = "button[aria-label='confirm']";
```

Comment:
All these seem to be generic which can be used by other tests as well . Please add in common.view.ts

Reply:
Moved